device/dhcp/clients/v6/dhcpv6client.yaml

```diff
@@ -0,0 +1,146 @@
+            iapd:
+              x-field-uid: 3
+            ianapd:
+              x-field-uid: 4
+        iana:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 2
+        iata:
+          $ref: '#/components/schemas/Device.Dhcpv6client.NoIaTimeValue'
+          x-field-uid: 3  
+        iapd:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 4
+        ianapd:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 5
+
+    Device.Dhcpv6client.IaTimeValue:
+      description: >-
+        The container for the suggested times at which the client contacts the server or any available server.
+      type: object
+      properties:
+        ta_1:
+          description: >-
+            The suggested time at which the client contacts the server from which the addresses were obtained to 
+            extend the lifetimes of the addresses assigned.
+          type: integer
+          format: uint64
+          maximum: 2147483647
+          default: 302400
+          x-field-uid: 1
+        ta_2:
+          description: >-
+            The suggested time at which the client contacts any available server to extend the lifetimes of the addresses assigned.
+          type: integer
+          format: uint64
+          maximum: 2147483647
+          default: 483840
+          x-field-uid: 2
+
+    Device.Dhcpv6client.NoIaTimeValue:
+      description: >-
+        The container where no suggested IA timeout are specified.
+      type: object
+
+    Device.Dhcpv6client.DuidType:
+      type: object
+      properties:
+        choice:
+          description: >-
+            Each DHCP client and server has a DUID. DHCP clients use DUIDs to identify a server in messages where a server 
+            .needs to be identified.
+          type: string
+          default: llt
+          x-field-uid: 1
+          x-enum:
+            llt:
+              x-field-uid: 1
+            en:
+              x-field-uid: 2
+            ll:
+              x-field-uid: 3
+        llt:
+          $ref: '#/components/schemas/Device.Dhcpv6client.NoDuid'
+          x-field-uid: 2
+        em:
+          $ref: '#/components/schemas/Device.Dhcpv6client.DuidValue'
+          x-field-uid: 3  
+        ll:
+          $ref: '#/components/schemas/Device.Dhcpv6client.NoDuid'
+          x-field-uid: 4
+
+    Device.Dhcpv6client.DuidValue:
+      description: >-
+        The container for the DUID-EN. This consists of the 4-octet vendor's registered Private Enterprise Number 
+        as maintained by IANA [IANA-PEN] followed by a unique identifier assigned by the vendor.
+      type: object
+      properties:
+        enterprise_id:
+          description: >-
+            4-octet vendor's registered Private Enterprise Number as maintained by IANA [IANA-PEN].
+          type: integer
+          format: uint64
+          minimum: 1
+          maximum: 2147483647
```

Comment:
This has a 31 bit max .. is this intended by protocol ?  If yes, can the format be uint32 ?

Reply:
updated